pages/mobile/login_page.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The convention in this directory is not to include the `_page` suffix on the filename. I know that the desktop directory is inconsistent, but I think we should follow the convention for the directory. I'll likely clean up the other page object names soon to match this.